tao_compiler/mlir/disc/tests/tensorflow_ops/data/squeeze_d_f32_2.mlir

```diff
@@ -0,0 +1,9 @@
+module attributes {tf.versions = {bad_consumers = [], min_consumer = 0 : i32, producer = 0 : i32}} {
+  func @main(%arg0: tensor<?x?x?xf32>) -> tensor<?x?xf32> attributes {tf.entry_function = {inputs = "{{INPUTS}}", outputs = "{{OUTPUTS}}", input_placements="{{INPUT_PLACEMENTS}}", output_placements="{{OUTPUT_PLACEMENTS}}"}} {
+    %graph = tf_executor.graph {
+      %1:2 = tf_executor.island wraps "tf.Squeeze"(%arg0) {T = f32, device = "", squeeze_dims = [-1]} : (tensor<?x?x?xf32>) -> tensor<?x?xf32>
```

Comment:
Nit: make the input as `tensor<?x?x1xf32>`

Reply:
It's the test case for fully dynamic shape. For partial dynamic shape case, we have another UT.